kafka-client-lib/src/main/java/io/jenkins/plugins/remotingkafka/exception/RemotingKafkaException.java

```diff
@@ -0,0 +1,11 @@
+package io.jenkins.plugins.remotingkafka.exception;
+
+public class RemotingKafkaException extends Exception {
+    RemotingKafkaException() {
+
+    }
+
+    RemotingKafkaException(String message) {
```

Comment:
Later you will likely need a `RemotingKafkaException(String message, Throwable cause)` constructor